# 0 - 100
uint8 ratio
---
bool success true
